script/tool/test/util.dart

```diff
@@ -85,14 +87,14 @@ Directory createFakePlugin(
     final Directory exampleDir = pluginDirectory.childDirectory('example')
       ..createSync();
     createFakePubspec(exampleDir,
-        name: '${name}_example', isFlutter: isFlutter);
+        name: '${name}_example', isFlutter: isFlutter, includeVersion: false, publish_to: 'none');
```

Comment:
While you're touching the utils, could you fix the naming of this parameter? It should be `publishTo`.

Reply:
done